web/src/main/java/org/springframework/security/web/FilterChainProxy.java

```diff
@@ -270,12 +282,15 @@ public void setFirewall(HttpFirewall firewall) {
 	/**
 	 * Sets the {@link RequestRejectedHandler} to be used for requests rejected by the
 	 * firewall.
-	 * @param requestRejectedHandler the {@link RequestRejectedHandler}
+	 * @param requestRejectedHandlers the {@link RequestRejectedHandler}
 	 * @since 5.2
 	 */
-	public void setRequestRejectedHandler(RequestRejectedHandler requestRejectedHandler) {
-		Assert.notNull(requestRejectedHandler, "requestRejectedHandler may not be null");
-		this.requestRejectedHandler = requestRejectedHandler;
+	public void setRequestRejectedHandlers(Collection<RequestRejectedHandler> requestRejectedHandlers) {
```

Comment:
Unfortunately, we cannot change the names of public methods. If using a composite implementation, this change isn't necessary anyway.

Reply:
Reverted